src/lib/extensions.js

```diff
@@ -48,8 +48,14 @@ export const CODE_EXTENSIONS = [
     'yaml',
 ];
 
+export const EXCEL_EXTENSIONS = ['xls', 'xlsm', 'xlsx'];
```

Comment:
Do we want to include `xlsb`? Should we update OfficeViewer to use parts of these constants?

Reply:
> Do we want to include `xlsb`? Should we update OfficeViewer to use parts of these constants?

`xlsb` conversion to pdf is not supported. But OfficeViewer supports `xlsb` but not `xls`...

Reply:
Should we add a comment so this nuance isn't lost?